db/migrate/20221128104415_create_roles.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRoles < ActiveRecord::Migration[6.0]
```

Comment:
this is not required. Manage user roles with UserRole model and user_roles db schema.
Create an association between User and UserRole.
Example: User.first.user_roles should give the user roles.